v2/emailpassword/advanced-customizations/apis-override/usage.mdx

```diff
@@ -75,4 +77,82 @@ SuperTokens.init({
 
 - `originalImplementation` and `originalImplementationEmailVerification` are objects that contain apis that have the original implementation for this and the email verification recipe. They can be used in your custom apis as a way to use the SuperTokens' default behaviour.
 - In the above code snippet, we override the `signUpPOST` api of this recipe. This api will be called when the user clicks the sign up button on the frontend.
-- Likewise, we override the `verifyEmailPOST` api from the email verification recipe.
\ No newline at end of file
+- Likewise, we override the `verifyEmailPOST` api from the email verification recipe.
+
+
+## Method 2: Modify the API code
```

Comment:
This heading is bad too. Even in method 1, you are modifying the API code.. Change this to "Method 2: Change API logic by copy / paste"